example-tracing/README.md

```diff
@@ -0,0 +1,76 @@
+# Example Tracing
+
+## Overview
+
+This example illustrates how to enable tracing for a service deployed in Kyma. For demostration, it creates a [go application](./order-front.go). This application uses [http-db-service](../http-db-service) for CRUD operations on orders.
+
+Refer to the following to understand configuration related to tracing
+
+1. [go application](./order-front.go) : 
+   - How trace headers are propagated?
+
+1. [example-tracing.yaml](./example-tracing.yaml)
+   - port naming 
+   - `app` label is configured? 
+
+## Prerequisites
+
+- A [Docker](https://docs.docker.com/install) installation
+- Kyma as the target deployment environment
+- An environment to which to deploy the example
+
+## Installation
+
+### Local installation
+
+1. Build the Docker image
+    
+	```bash
+    docker build . -t order-front:latest
+    ```
+
+1. Run the recently built image
+    
+	```bash
+    docker run -p 8080:8080 order-front:latest
+    ```
+
+### Cluster installation
+
+1. Export your environment as a variable by replacing the `<environment>` placeholder in the following command and running it:
+    
+	```bash
+    export KYMA_EXAMPLE_ENV="<environment>"
+    ```
+
+1. Change the `hostname: order-front-api.kyma.local` to `hostname: order-front-api.{domain-of-kyma-cluster}` in [example-tracing.yaml](./example-tracing.yaml)
+
+1. Deploy the service
+    
+	```bash
+    kubectl apply -f ./example-tracing.yaml -n $KYMA_EXAMPLE_ENV
+    ```
+
+### Test the service
```

Comment:
This should be a separate point as this works for both cluster and local deployments. Additionally, this should be the part that gets you to the main point of the example which, in this case, I think is tracing. Why would anyone execute this step? To get traces.   

`### Test the service` -> `## Get traces from the example service`